cmd/ping_test.go

```
@@ -434,6 +441,58 @@ func Test_Execute_Ping_Infinite(t *testing.T) {
+	expectedOpts := createDefaultMeasurementCreate("ping")
+	expectedOpts.Options.Packets = 16
+	expectedOpts.InProgressUpdates = true
+
+	expectedResponse := createDefaultMeasurementCreateResponse()
+	expectedMeasurement := createDefaultMeasurement("ping")
+
+	gbMock := apiMocks.NewMockClient(ctrl)
+	gbMock.EXPECT().CreateMeasurement(gomock.Any(), expectedOpts).Return(expectedResponse, nil)
+	var runCtx context.Context
+	gbMock.EXPECT().GetMeasurement(gomock.Any(), measurementID1).DoAndReturn(func(ctx context.Context, _ string) (*globalping.Measurement, error) {
+		runCtx = ctx
```

golangci-lint (2.12.2)

[error] nested context in function literal

(fatcontext)